website/components/navigation/Navigation.tsx

```diff
@@ -19,57 +23,73 @@ const logoStyles = css`
 `;
 
 function Navigation() {
+  const viewport = useViewportSize();
+  const isMobile = viewport && viewport.width < 768;
+  const Group = isMobile ? MobileNavigationGroup : SideNavGroup;
+  const Item = isMobile ? MobileNavigationItem : SideNavItem;
+
+  const groupProps =
```

Comment:
prefer using ternary operator here so that `groupProps` can't be `false`

Reply:
👍